app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -217,6 +195,54 @@ class AttendeeFragment : Fragment() {
 
         attendeeViewModel.initializeSpinner()
 
+        rootView.cardNumber.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable?) {
+            }
+
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {
+            }
+
+            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+                val visaPattern = Pattern.compile("^4[0-9]{0,15}$")
+                val masterCardPattern = Pattern.compile("^(5[1-5]|222[1-9]|22[3-9][0-9]|2[3-6]" +
+                    "[0-9]{2}|27[01][0-9]|2720)[0-9]{0,15}$")
+                val americanExpressPattern = Pattern.compile("^3[47][0-9]{0,15}$")
```

Comment:
Stripe most probably has pre defined functions to check the above. Can you confirm? if yes use them

Reply:
They have their own cardInputwidget which handles all these operation https://stripe.com/docs/mobile/android .

- That would mean changing the whole input section .. should I use it ???

Reply:
If that is the case then it is fine

Reply:
Can you elaborate is current commit is fine or I should use cardInputWidget

Reply:
@nikit19 please reply

Reply:
It is fine